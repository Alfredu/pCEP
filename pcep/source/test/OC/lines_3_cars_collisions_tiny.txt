detect_obj_loc+avg.dolce
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 9.78029999999998 float t 14770104522000
1 obj_loc int obj_id 2 string type L float loc_x 9.51219999999998 float loc_y 0.5803 float t 14770104522000
1 obj_loc int obj_id 3 string type L float loc_x 16.8122000000002 float loc_y 0.3803 float t 14770104522000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 9.88029999999998 float t 14770104523000
1 obj_loc int obj_id 2 string type L float loc_x 9.61219999999998 float loc_y 0.5803 float t 14770104523000
1 obj_loc int obj_id 3 string type L float loc_x 16.6122000000002 float loc_y 0.3803 float t 14770104523000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 9.98029999999998 float t 14770104524000
1 obj_loc int obj_id 2 string type L float loc_x 9.71219999999998 float loc_y 0.5803 float t 14770104524000
1 obj_loc int obj_id 3 string type L float loc_x 16.4122000000002 float loc_y 0.3803 float t 14770104524000
1 obj_loc int obj_id 1 string type L float loc_x 0.3122 float loc_y 10.0803 float t 14770104525000
1 obj_loc int obj_id 2 string type L float loc_x 9.81219999999998 float loc_y 0.5803 float t 14770104525000
1 obj_loc int obj_id 3 string type L float loc_x 16.2122000000002 float loc_y 0.3803 float t 14770104525000
