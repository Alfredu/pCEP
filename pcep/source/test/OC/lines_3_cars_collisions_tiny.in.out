2 collision_with_moving_object float t 14770104172544.000000 float loc_y 9.880300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 9.880300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 9.980300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 9.980300
2 collision_with_moving_object float t 14770104172544.000000 float loc_y 10.080300 float loc_x 0.312200 string type "L" int obj_id 1 float loc_y 10.080300
7 Shutdown int Result 0 
